mmrazor/models/mutables/mutable_module/one_shot_mutable_module.py

```diff
@@ -63,37 +63,37 @@ def forward(self, x: Any) -> Any:
             return self.forward_choice(x, choice=self.current_choice)
 
     @abstractmethod
-    def sample_choice(self) -> CHOICE_TYPE:
+    def sample_choice(self) -> str:
```

Comment:
The ``sample_choice`` does not need to pass in any args in oneshot, and it needs to pass in the arch param in diff. If it is placed in the base class, it will be overloaded in the diff, which will bring some confusion.